src/components/side-menu/list_item-test.html

```diff
@@ -11,4 +11,4 @@
   <span class="menuText">Sleep Timer</span>
 </li>
 <!-- zur Visualisierung -->
-<li class="menu_list_item" style="height: 150px"></li>
+<li class="menu_list_item" style="height:150px"></li>
```

Comment:
Inline styles are a little outdated in my opinion, I definitely don't expect them here since I see you using CSS files. However I did spot that little comment up there, is this to justify the inline style? :D

Reply:
ah, that was only for testing, jap, i forgot to delete it

Reply:
solved